sites/all/themes/orwell/sass/component/_dialog.scss

```diff
@@ -128,3 +128,21 @@
 .ui-widget-overlay {
   opacity: .4;
 }
+
+// Specific dialogs.
+
+#ding-reservation-delete-reservations-form {
```

Comment:
ID's should be avoided as codeclimate rightfully says.
Forms usually already have a similar class out of the box.

Reply:
Yes, but sadly not this one. This is what I have to work with:

![selection_165](https://user-images.githubusercontent.com/229422/32777497-9b5ed1d2-c936-11e7-8e0b-5038da319136.png)